bloom-housing/bloom
# feat: save and continue

This PR addresses #3654 


## Description

This PR changes the "Save & Exit" button so it just reads "Save", and clicking it will save the listing and remain in the edit state. In addition, the "Cancel" button has been renamed "Exit" and it displays a prompt to the user to either save or discard changes.

## How Can This Be Tested/Reviewed?

Try saving or exiting listings in various stages (new, unpublished, published, etc.) and verify the behavior matches the AC/Figma.

## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
